glib-macros/tests/clone.rs

```diff
@@ -4,6 +4,169 @@ use std::rc::Rc;
 
 use glib::clone;
 
+#[test]
+#[cfg(feature = "unstable-clone-syntax")]
```

Comment:
These tests should probably continue to test the old implementation. Also tests/closure.rs

Otherwise looks good to me